### PR TITLE
(docs) Add release notes for 7.0.0

### PR DESCRIPTION
**Pull request by Zak-Kent**

### CODE REVIEW

File: documentation/release_notes/release_notes_latest.markdown [hunk 1]
```diff
@@ -21,6 +21,31 @@ canonical: "/puppetdb/latest/release_notes.html"
...
 
+
+## PuppetDB 7.0.0
+
+Released 19 November 2020
+
+### Bug fixes
+
+- PuppetDB no longer retries queries internally, suppressing some transient
+  connection errors. Instead, it will return an error code more immediately.
```
Comment (clairecadman) [inline documentation/release_notes/release_notes_latest.markdown:32]:
```suggestion
  connection errors. Instead, it immediately returns an error code.
```

File: documentation/release_notes/release_notes_latest.markdown [hunk 2]
```diff
@@ -21,6 +21,31 @@ canonical: "/puppetdb/latest/release_notes.html"
...
+  For the moment, the previous behavior can be restored by setting the
```
Comment (clairecadman) [inline documentation/release_notes/release_notes_latest.markdown:33]:
```suggestion
You can restore the previous behaviour by setting the
```

File: documentation/release_notes/release_notes_latest.markdown [hunk 3]
```diff
@@ -21,6 +21,31 @@ canonical: "/puppetdb/latest/release_notes.html"
...
+  `PDB_USE_DEPRECATED_QUERY_STREAMING_METHOD` environment variable. See the
+  [configuration information](https://puppet.com/docs/puppetdb/latest/configure.html#experimental-environment-variables)
+  for further details.
+
+- PuppetDB won't hold an extra database connection open while generating query
+  responses. Previously it would create and hold an extra connection open during
+  the initial phase of the response. For the moment, the previous behavior can
```
Comment (clairecadman) [inline documentation/release_notes/release_notes_latest.markdown:40]:
```suggestion
  the initial phase of the response. You can restore the previous behavior
```

File: documentation/release_notes/release_notes_latest.markdown [hunk 4]
```diff
@@ -21,6 +21,31 @@ canonical: "/puppetdb/latest/release_notes.html"
...
+  be restored by setting the `PDB_USE_DEPRECATED_QUERY_STREAMING_METHOD` environment
```
Comment (clairecadman) [inline documentation/release_notes/release_notes_latest.markdown:41]:
```suggestion
by setting the `PDB_USE_DEPRECATED_QUERY_STREAMING_METHOD` environment
```

**Review by clairecadman — APPROVED** [review]
I made a couple of suggestions, but otherwise looks good.

**Comment by Zak-Kent** [comment]
Thanks @clairecadman!

**Pull request merged**